blinds/blinds_api.py

```diff
@@ -88,10 +100,16 @@ def __init__( self, blinds, blindsSchedule, temperatureHandler ):
     '''
     def getTemperature( self ):
         print( "processing request for GET temperature")
-        
+
+        # take a single reading and return a compensated_reading object
+        sample = bme280.sample(self.bus, self.tempSensorAddress, self.calibrationParams)
+
+        # get the temperature attribute from the compensated_reading class 
+        int_temp = sample.temperature
+
         # dummy temporary data
         data = {
-            "temperature" : "20",
+            "temperature" : str(int_temp),
             "temp_units" : "C"
         }
```

Comment:
I might suggest potting all these operations in a try-except block, even if we leave the exceptions unhandled for now. As an example, the one i've been using as a temp solution is
```
    def postSchedule( self, schedule ):
        print( "processing request for POST schedule")
        print( "received schedule=\n", schedule )

        try:
            self._blindsSchedule = BlindsSchedule.fromJson( schedule )
            return schedule, RESP_CODES[ "ACCEPTED" ]

        except ( InvalidBlindsScheduleException, BlindSchedulingException ) as err:
            return str( err ), RESP_CODES[ "BAD_REQUEST" ]

        except Exception as err: # catch all others and return an error message 
            #TODO : More specialized handling for safety, we don't want just any error messages being spit to the user, for 
            # now, in the testing phase we return the error 
            return str( err ), RESP_CODES[ "BAD_REQUEST" ]
```
Of course you can put your own errors to catch if you have some issues you already want to deal with, otherwise just keep the catch all at the bottom with the TODO until we figure out a cleaner way of handling other unexpected stuff rather than returning it to the user.